service-swagger-fragment-os/src/main/java/com/ibm/watson/solutions/api/listeners/SwaggerSetup.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * (C) Copyright IBM Corp. 2001, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.watson.solutions.api.listeners;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.Path;
+
+import com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule;
+import com.ibm.watson.solutions.api.rest.CustomSwaggerApiListingResource;
+import com.ibm.watson.solutions.api.rest.CustomSwaggerApiListingResource2;
+
+import io.swagger.jaxrs.config.BeanConfig;
+import io.swagger.jaxrs.listing.ApiListingResource;
+import io.swagger.jaxrs.listing.SwaggerSerializers;
+import io.swagger.util.Json;
+
+/**
+ *
+ * Helper methods to initialize the swagger environment
+ *
+ */
+public class SwaggerSetup  {
+
+	static{
+        //register jaxb annotation module as secondary annotation handler
+		JaxbAnnotationModule jaxbAnnotationModule = new JaxbAnnotationModule();
```

Comment:
I'm seeing mixed whitespace in these.  Would prefer to see it cleaned up - tangentially, I think we did something to align between mac and windows whitespace in our other project, should take a look to see if that's worthwhile here.

Reply:
Agreed, will clean this up.

Reply:
A quick pass over the repo pointed me towards the following files as having DOS line endings.
```
$ find . -exec file {} \; | sort -u | grep CRLF
./service-base-utils-bld-os/pom.xml: exported SGML document text, ASCII text, with CRLF, LF line terminators
./service-base-utils-parent-os/pom.xml: exported SGML document text, ASCII text, with CRLF, LF line terminators
./service-encryption-os/pom.xml: exported SGML document text, ASCII text, with CRLF, LF line terminators
```